core-web/libs/portlets/edit-ema/portlet/src/lib/dot-ema-shell/store/dot-ema.store.ts

```diff
@@ -259,7 +257,27 @@ export class EditEmaStore extends ComponentStore<EditEmaState> {
                 iframeURL,
                 isEnterpriseLicense,
                 state: currentState,
-                dragItem
+                dragItem,
+                showContentletTools:
+                    editorData.canEditVariant &&
+                    !!contentletArea &&
+                    !editorData.device &&
+                    editor.page.canEdit &&
+                    (currentState === EDITOR_STATE.IDLE ||
+                        currentState === EDITOR_STATE.DRAGGING) &&
+                    !editorData.page.isLocked,
+                showDropzone:
+                    editorData.canEditVariant &&
+                    !editorData.device &&
+                    (currentState === EDITOR_STATE.DRAGGING ||
+                        currentState === EDITOR_STATE.SCROLLING),
```

Comment:
do we want to show the dropzone while scrolling?

Reply:
Should we check the bounds here?

Reply:
Yes, we need to have the dropzone
If the dropzone is hidden, the dragover is lost
The dropzone is rendered when scrolling but with empty bounds array

For that, is not needed check bounds, because with the states we already know that (Dragging = normal bounds, scrolling = empty array bounds)